.eslintrc.js

```diff
@@ -6,7 +6,13 @@ module.exports = {
   extends: ['plugin:vue/essential', '@vue/prettier'],
   rules: {
     'no-console': 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    'prettier/prettier': [
+      'error',
+      {
+        singleQuote: true
```

Comment:
When we push upstream we should add comments for *why* we are doing things like this.

Reply:
I agree and also feel we should have documentation for all the best practices.

Reply:
agreed.